_pytest/assertion/__init__.py

```diff
@@ -113,7 +115,8 @@ def callbinrepr(op, left, right):
         for new_expl in hook_result:
             if new_expl:
                 if (sum(len(p) for p in new_expl[1:]) > 80*8
-                        and item.config.option.verbose < 2):
+                        and item.config.option.verbose < 2
+                        and 'CI' not in os.environ):
```

Comment:
Cool. :smile:

I think then we should have a check similar to what you did in [pytest-vw](https://github.com/The-Compiler/pytest-vw/blob/master/pytest_vw.py#L13). If some system is left out, we can just add it later. I like the idea of supporting most/all systems out of the box.